### ISSUE TITLE
Module does not report errors on GAE Flex

### ISSUE THREAD

**Issue by steren** [GoogleCloudPlatform/cloud-errors-nodejs#28]
Here is how to reproduce the problem:

* I enabled the Error Reporting API.
* I deploy the following code to GAE flex  (based on the GAE flex node hello world)
* I open "/crash" and "/error"

Observed: No error appear in Error Reporting UI
Expected: I see 2 error groups in Error Reporting UI

I also tested by explicitly providing projectId in the initialization, I had the same results: no error.

```JS
'use strict';
var express = require('express');
var app = express();
var errorHandler = require('@google/cloud-errors')();

// Say hello!
app.get('/', function (req, res) {
  res.status(200).send('Hello, world!');
});

app.get('/crash', function (req,res) {
  JSON.parse("{\"malformedJson\": true");
});
app.get('/error', function(req,res, next) {
    // You can push in errors manually
    res.send("Error");
    res.end();
    next(new Error("Got traffic on the errorRoute"));
});
if (module === require.main) {
  var server = app.listen(process.env.PORT || 8080, function () {
    var port = server.address().port;
    console.log('App listening on port %s', port);
  });
}
module.exports = app;
```

**Comment by steren** [comment]
Note that the issue is not https://github.com/GoogleCloudPlatform/cloud-errors-nodejs/issues/29 since I also tested by explicitly giving the projectId

**Comment by steren** [comment]
Maybe it is due to the fact that I cannot see a GOOGLE_APPLICATION_CREDENTIALS env variable when I print the content of `process.env`?

I do not expect to have to care about authentication on GAE Flex.

**Comment by cristiancavalli** [comment]
@steren I'll try this today, but in the example provided it doesn't seem like you ever declared `app.use(errorHandler.express)`

**Comment by steren** [comment]
Arg thanks for spotting, I mixed up.
Let me try again.

**Comment by steren** [comment]
Actually it's still not a success after adding the `app.use` line:

I don't see errors

```JS
'use strict';
var express = require('express');
var app = express();
var errorHandler = require('@google/cloud-errors')({projectId: 'test-gae-flex-node-module'});
// [START hello_world]
// Say hello!
app.get('/', function (req, res) {
  res.status(200).send(JSON.stringify(process.env, null, '\t'));
});
// [END hello_world]
app.get('/crash', function (req,res) {
  JSON.parse("{\"malformedJson\": true");
});
app.get('/error', function(req,res, next) {
    // You can push in errors manually
    res.send("Error");
    res.end();
    next(new Error("Got traffic on the errorRoute"));
});
app.use(errorHandler.express);
if (module === require.main) {
  // [START server]
  // Start the server
  var server = app.listen(process.env.PORT || 8080, function () {
    var port = server.address().port;
    console.log('App listening on port %s', port);
  });
  // [END server]
}
module.exports = app;
```

**Comment by steren** [comment]
I tested by using the code at HEAD, and I confirm that the above code reports errors corretcly.
(Even when not providing a projectId in the initialization)
